internal/component/poddistruptionbudget/poddisruptionbudget.go

```diff
@@ -107,6 +113,12 @@ func buildResource(etcd *druidv1alpha1.Etcd, pdb *policyv1.PodDisruptionBudget)
 	pdb.Spec.Selector = &metav1.LabelSelector{
 		MatchLabels: druidv1alpha1.GetDefaultLabels(etcd.ObjectMeta),
 	}
+	if metav1.HasAnnotation(etcd.ObjectMeta, annotationAllowUnhealthyPodEviction) {
+		alwaysAllow := policyv1.AlwaysAllow
+		pdb.Spec.UnhealthyPodEvictionPolicy = &alwaysAllow
```

Comment:
You can use `utils.PointerOf` instead, then you do not need to declare `alwaysAllow`

Reply:
Thanks for the suggestion. Addressed now